i18n/zh/docusaurus-plugin-content-docs/current/architecture/cloud/device_controller.md

```diff
@@ -1,71 +1,62 @@
 ---
-title: Device Controller
+title: 设备控制器
 sidebar_position: 3
 ---
-## Device Controller Overview
- The device controller is the cloud component of KubeEdge which is responsible for device management. Device management in KubeEdge is implemented by making use of Kubernetes
- [Custom Resource Definitions (CRDs)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) to describe device metadata/status and a device controller to synchronize these devices updates between edge and cloud.
- The device controller starts two separate goroutines called `upstream controller` and `downstream controller`. They are not separate controllers but are just named here for clarity.
 
+## 设备控制器(Device Controller)概述
 
+Device Controller是KubeEdge的云端组件，负责设备管理。在KubeEdge中，设备管理是通过使用Kubernetes的[自定义资源定义 (CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)来描述设备的元数据/状态，并通过Device Controller在Edge node 和 Cloud之间同步这些设备更新来实现的。Device Controller启动了两个独立的协程，分别称为 **`上行控制器(upstream controller)`** 和 **`下行控制器(downstream controller)`**。它们并不是独立的控制器，只是为了便于理解而命名。
 
-The device controller leverage the device model and device instance to implement device management :
- - **Device Model**: A `device model` describes the device properties exposed by the device and property visitors to access these properties. A device model is like a reusable template that is used to create and manage different kinds of devices.
- Details on the device model definition can be found [here](https://github.com/kubeedge/kubeedge/tree/master/docs/proposals/device-crd.md#device-model-type-definition).
- - **Device Instance**: A `device` instance represents an actual device object. It is like an instantiation of the `device model` and references properties defined in the model. The device spec is static while the device status contains dynamically changing like the desired state of a device property and the actual state reported by the device.
- Details on device instance definition can be found [here](https://github.com/kubeedge/kubeedge/tree/master/docs/proposals/device-crd.md#device-instance-type-definition).
+Device Controller利用**设备模型(Device Model)**和**设备实例(Device Instance)**来实现设备管理：
 
- **Note**: A sample of the device model and device instance for a few protocols can be found at $GOPATH/src/github.com/kubeedge/kubeedge/build/crd-samples/devices
+- **Device Model**：**`Device Model`**描述了设备公开的设备属性以及访问这些属性的属性访问者。设备模型类似于可重复使用的模板，用于创建和管理不同类型的设备。其中包括属性访问者的三种访问模式，分别是OpcUA, Modbus和Bluetooth,设备模型定义的详细信息可以在[这里](https://github.com/kubeedge/kubeedge/tree/master/docs/proposals/device-crd.md#device-model-type-definition)找到。
+  
```

Comment:
```suggestion
- **Device Model**：**`Device Model`**描述了设备公开的设备属性以及访问这些属性的属性访问者。Device Model类似于可重复使用的模板，用于创建和管理不同类型的设备。Device Model定义的详细信息可以在[这里](https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/device-crd-v1beta1.md#device-model-type-definition)找到。
  
```

Inaccuracies in English documents can be updated simultaneously

Reply:
Modified in both Chinese version and English